src/engine/strat_engine/thinpool/filesystem.rs

```diff
@@ -291,6 +291,22 @@ impl StratFilesystem {
 }
 
 impl Filesystem for StratFilesystem {
+    fn send_udev_change(&self) -> StratisResult<()> {
```

Comment:
We already do these actions, slightly differently, in devlinks.rs. Can you factor that out, since it becomes a shared action now, with this PR?

Also, did you decide against only triggering the rule on a change event? That still seems sensible to me, to avoid an initial failure of the script.

Reply:
I decided against only doing it on a change event because `mkfs.xfs` will cause a change event even on filesystem creation so we should potentially ask the question of whether we should be logging at any level higher than debug so that people can filter it out if they're not actively debugging a problem.

Reply:
Ok! We decided in standup that the best thing would be to reduce the severity of the messages from the udev script to ```debug```.

Reply:
@mulkieran I'm not sure that this is actually a shared action just because the only shared piece is really writing to the udev file. The way the path to the udev file is generated is entirely different. Do you want me to just factor out the `.write_all("change".as_bytes())` call?

Reply:
> @mulkieran I'm not sure that this is actually a shared action just because the only shared piece is really writing to the udev file. The way the path to the udev file is generated is entirely different. Do you want me to just factor out the `.write_all("change".as_bytes())` call?

I see your point. And it's a toss-up whether just the writing part is worth abstracting yet. Let's wait until the next time we find a need to generate a synthetic event to worry about it.